src/Discord.Net.WebSocket/Entities/Interaction/Context Menu Commands/User Commands/SocketApplicationUserCommandData.cs

```diff
@@ -6,7 +6,7 @@
 namespace Discord.WebSocket
 {
     /// <summary>
-    ///     Represents the data tied with the <see cref="SocketSlashCommand"/> interaction.
+    ///     Represents the data tied with the <see cref="SocketApplicationUserCommand"/> interaction.
     /// </summary>
     public class SocketApplicationUserCommandData : SocketEntity<ulong>, IApplicationCommandInteractionData
```

Comment:
Shouldn't this inherit from BaseData? Or is the purpose of that class different?

Reply:
slightly different. one is an interface, the other is a class to support the SocketCommandBase methods.